src/components/tooltip/index.js

```diff
@@ -0,0 +1,10 @@
+import { Tooltip as MuiTooltip } from "@mui/material";
+
+export const Tooltip = ({ children, title, ...props }) => (
+    <MuiTooltip
+        title={<h1 style={{ fontSize: "12px" }}>{title}</h1>}
```

Comment:
please move styles into the corresponding .scss file and use `size.$text` / `size.$large-text` or similar dynamic sized instead of hard coding it.